internal/lockfilescalibr/language/dotnet/nugetpackagelock/extractor-v1_test.go

```diff
@@ -1,113 +1,113 @@
-package lockfilescalibr_test
+package nugetpackagelock_test
```

Comment:
Thinking more, is the convention of "<language>/<pkg manager>" something we definitely need to keep?

There are some edge cases. e.g. https://en.wikipedia.org/wiki/Conda_(package_manager), which is a package manager for R **and** python